crates/starknet-os/src/hints/tests.rs

```diff
@@ -278,4 +282,136 @@ pub mod tests {
             );
         }
     }
+
+    #[test]
+    fn test_fill_holes_in_rc96_segment() {
+        let mut vm = VirtualMachine::new(false);
+        vm.set_fp(1);
+        vm.add_memory_segment();
+        vm.add_memory_segment();
+
+        let mut exec_scopes = ExecutionScopes::new();
+        let ids_data = ids_data![vars::ids::RANGE_CHECK96_PTR];
+        let ap_tracking = ApTracking::default();
+        let constants = HashMap::new();
+
+        let mut rc96_segment = vm.add_memory_segment();
+        rc96_segment.offset = 10;
```

Comment:
Minor suggestion: define a constant for 10 and use it throughout the test, I think that is clearer than reusing `rc96_segment.offset` in `for` loops below.